### DISCUSSION TITLE
Small holes in outer wall

### DISCUSSION THREAD

**Discussion by Hoeby** [smartavionics/Cura#240] [Q&A]
I am running version 4.20.15-mb and there are a lot of features i was looking for, great work.

The only thing where i am running in, is that there are small holes in the outer wall.
They are not on the seam, but random.

Tried the follow items, without luck:
- Increase temperature
- Increase flow settings
- Disable "filter out tiny gaps"
- Lower the retract settings, but then stringing occures

It got better, but not solved.
Made a test with Cura 5.4.0, then i don't have the holes. This is on the same printer, same filament and temperature. Even with lower flow settings.

Googled it and most of the people say it is because of the retraction setting.
This setting is the samen in original cura en de smartavionics version. 

Can't find the setting which causes this, does somebody has a clue?
This is not my cube, but the same result
 <img src="https://content.invisioncic.com/ultimake/monthly_2019_01/20190116_183721.jpg.d7be256f90edcdf3d375383dd2a33602.jpg" width="300" >

**Comment by smartavionics** [comment]
Hi, as you say, holes like that are often caused by incorrect retraction settings. Specifically, if too much retraction is used, it sucks air into the nozzle and that air comes out at random locations making a zit but also it makes a noticeable popping sound. If you can hear little pops while it is printing, that's the reason. The solution is to reduce the amount of retraction in use. Are you using a direct drive or bowden extruder? How much retraction are you using?

**Comment by Hoeby** [comment]
Running a direct drive. My retraction is 0.8mm with a speed of 30mm/s
